src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -748,10 +759,22 @@ func forceEOF(yylex interface{}) {
 
 %type	<setExprs>
 	set_list
+	txn_list
```

Comment:
The name `txn_list` is not so good, and can `txn_list` combined to `set_list` as they both start with `set`.